app/controllers/find/results_controller.rb

```diff
@@ -11,7 +11,7 @@ def index
 
       @results_view = ResultsView.new(query_parameters: matched_params)
       @filters_view = ResultFilters::FiltersView.new(params: matched_params)
-      @courses = @results_view.courses.page params[:page]
+      @pagy, @courses = pagy(@results_view.courses, page: params[:page])
```

Comment:
I think the `page: params[:page]` is implicit for Pagy.

```suggestion
      @pagy, @courses = pagy(@results_view.courses)
```